api/kyverno/v1/common_types.go

```diff
@@ -313,6 +314,46 @@ type Validation struct {
 	// Deny defines conditions used to pass or fail a validation rule.
 	// +optional
 	Deny *Deny `json:"deny,omitempty" yaml:"deny,omitempty"`
+
+	// PodSecurity applies exemptions for Kubernetes Pod Security admission
+	// by specifying exclusions for Pod Security Standards controls.
+	// +optional
+	PodSecurity *PodSecurity `json:"podSecurity,omitempty" yaml:"podSecurity,omitempty"`
+}
+
+type PodSecurity struct {
+	// Level defines the Pod Security Standard level to be applied to workloads.
+	// Allowed values are privileged, baseline, and restricted.
+	// +kubebuilder:validation:Enum=privileged;baseline;restricted
+	Level api.Level `json:"level,omitempty" yaml:"level,omitempty"`
+
+	// Version defines the Pod Security Standard versions that Kubernetes supports.
+	// Allowed values are v1.19, v1.20, v1.21, v1.22, v1.23, v1.24, v1.25, latest. Defaults to latest.
+	// +kubebuilder:validation:Enum=v1.19;v1.20;v1.21;v1.22;v1.23;v1.24;v1.25;latest
+	// +optional
+	Version string `json:"version,omitempty" yaml:"version,omitempty"`
+
+	// Exclude specifies the Pod Security Standard controls to be excluded.
+	Exclude []PodSecurityStandard `json:"exclude,omitempty" yaml:"exclude,omitempty"`
+}
+type PodSecurityStandard struct {
```

Comment:
Please add a comment here

Reply:
Missed it!

@ToLToL - I'd like to get this merge and test locally. Can you send a follow-up PR to add the description?